selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -15,39 +15,70 @@
 import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.LOADER_TIMEOUT_SEC;
 import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC;
 import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.REDRAW_UI_ELEMENTS_TIMEOUT_SEC;
+import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.WIDGET_TIMEOUT_SEC;
 import static org.openqa.selenium.support.ui.ExpectedConditions.invisibilityOfElementLocated;
 import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOf;
 import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
 
 import com.google.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
 import org.eclipse.che.selenium.core.SeleniumWebDriver;
 import org.eclipse.che.selenium.pageobject.dashboard.Dashboard;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 public class Workspaces {
   private final SeleniumWebDriver seleniumWebDriver;
+  private final WebDriverWait redrawUiElementsTimeout;
   private final Dashboard dashboard;
 
   @Inject
   public Workspaces(SeleniumWebDriver seleniumWebDriver, Dashboard dashboard) {
     this.seleniumWebDriver = seleniumWebDriver;
+    this.redrawUiElementsTimeout =
+        new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
     this.dashboard = dashboard;
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
   private interface Locators {
     String WORKSPACES_LIST = "//ng-transclude[@class='che-list-content']";
-    String WORKSPACE_ITEM_NAME =
-        "//div[contains(@class, 'workspace-name-clip')]//div[contains(@data-str, '%s')]";
-    String TOOLBAR_TITLE_NAME =
-        "//div[contains(@class,'che-toolbar')]//span[contains(text(),'%s')]";
-    String ADD_WORKSPACE_BTN =
-        "//a[contains(@ng-href, 'create-workspace')]/span[text()='Add Workspace']";
+    String TOOLBAR = "Workspaces";
+    String DOCUMENTATION_LINK = "//a[@ng-href='/docs/devops/intro/index.html']";
+    String ADD_WORKSPACE_BTN = "//che-button-primary[@che-button-title='Add Workspace']";
+    String DELETE_WORKSPACE_BTN = "//che-button-primary[@che-button-title='Delete']";
```

Comment:
@SkorikSergey 
this is the 'Delete' button, you may use the existing class or add any necessary attribute as well
https://github.com/eclipse/che/blob/03e1969339490799dddcd330f3ff4874e2a61b5b/dashboard/src/components/widget/list/che-list-header.html#L31-L35